packages/design/src/DataTable/Table.jsx

```diff
@@ -170,6 +171,18 @@ const TextCell = props => {
   return <Cell {...rest}>{data[rowIndex][columnKey]}</Cell>;
 };
 
+const LabelCell = props => {
```

Comment:
This component requires that an object must have a `tags` field which does not look to be generic enough to be part of the Table. Instead you could declare a function like below:

```ts
export function renderLabelCell(labels: []string){
 const $labels = tags.map(label => (
    <Label mb="1" mr="1" key={label} kind="secondary">
      {`${label}`}
    </Label>
  ));

  return <Cell>{$labels}</Cell>;  
}
```

And then use it like so in various places. 

```tsx
function LabelCell(props){
 const { rowIndex, data } = props;
 const { tags } = data[rowIndex];
 return renderLabelCell(tags);
}
```

Also, reasonable code duplication is OK :)

Reply:
Good suggestion, done :+1: